eitprocessing/features/breath_detection.py

```diff
@@ -82,7 +85,19 @@ def find_breaths(self, sequence: Sequence, continuousdata_label: str) -> Interva
         Returns:
             A list of Breath objects.
         """
-        continuous_data = sequence.continuous_data[continuousdata_label]
+        msg = "find_breaths expects a Sequence or ContinuousData object as first argument."
+        raise NotImplementedError(msg)
+
+    @find_breaths.register(Sequence)
+    def _(self, sequence: Sequence, continuousdata_label: str) -> IntervalData:
+        cd = sequence.continuous_data[continuousdata_label]
```

Comment:
Maybe to make it more clear that this is also continuous_data?
```suggestion
        continuous_data = sequence.continuous_data[continuousdata_label]
```